add-on/manifest.common.json

```diff
@@ -29,15 +30,24 @@
     "page": "dist/options/options.html"
   },
   "web_accessible_resources": [
-    "icons/png/ipfs-logo-off_19.png",
-    "icons/png/ipfs-logo-off_38.png",
-    "icons/png/ipfs-logo-off_128.png",
-    "icons/ipfs-logo-on.svg",
-    "icons/ipfs-logo-off.svg",
-    "dist/recovery/recovery.css",
-    "dist/recovery/recovery.html",
-    "dist/recovery/recovery.js"
+    {
+      "resources": [
+        "icons/png/ipfs-logo-off_19.png",
+        "icons/png/ipfs-logo-off_38.png",
+        "icons/png/ipfs-logo-off_128.png",
+        "icons/ipfs-logo-on.svg",
+        "icons/ipfs-logo-off.svg",
+        "dist/recovery/recovery.css",
+        "dist/recovery/recovery.html",
+        "dist/recovery/recovery.js"
+      ],
+      "matches": [
+        "<all_urls>"
+      ]
+    }
```

Comment:
I took a look at https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/ and the first line 

> Web-accessible resources are files inside an extension that can be accessed by web pages or other extensions. Extensions typically use this feature to expose images or other assets that need to be loaded in web pages, but any asset included in an extension's bundle can be made web accessible.

It appears that this is needed so that webpages can test if companion is being used, and if so, they can load recovery.html/js/css. Is that correct? are there other usecases where this is needed?